facade/service.go

```diff
@@ -1335,11 +1389,6 @@ func (f *Facade) verifyServiceForUpdate(ctx datastore.Context, svc *service.Serv
 		return err
 	}
 
-	err = svc.ValidEntity()
```

Comment:
Seems like it makes more sense to keep the call to `svc.ValidEntity()` here and remove the one at lines 1410-14142, because if the basic attributes of the service are bad, why do more extensive validation like child service, endpoints, etc?